datalad/distributed/create_sibling_ria.py

```diff
@@ -78,32 +78,36 @@ class CreateSiblingRia(Interface):
     The store's base path is expected to not exist, be an empty directory,
     or a valid RIA store.
 
+    Notes
+    -----
+
+
     RIA URL format
```

Comment:
so there can be no subsection whatsoever?

could/should there be explicit formatting, e.g.
```suggestion
    **RIA URL format**
```
?

Reply:
I tried `^^^` and it still complained about "Unexpected section title" :( Napoleon [docs](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) don't talk about subsections.

Nothing wrong about bolding the headers - I simply replicated what @mih used in his longer docstrings in -next. Both ways look fine to me.